.changeset/smooth-walls-grin.md

```diff
@@ -0,0 +1,5 @@
+---
+"@nacelle/nacelle-js": patch
+---
+
+feat(breaking!): cartLines now require correct shopify params.  Incorrect params will now throw an error
```

Comment:
Was there a reason incorrect params wouldn't throw before? They were still getting passed to the actual fetch call